otherlibs/stdune-unstable/top_closure.ml

```diff
@@ -1,34 +1,34 @@
-module Make (Keys : Top_closure_intf.Keys) (Monad : Monad_intf.S1) = struct
-  open Monad
+module Make (Keys : Top_closure_intf.Keys) (Monad : Monad_intf.S) = struct
```

Comment:
I guess, you could also do this?

```suggestion
module Make (Keys : Top_closure_intf.Keys) (M : Monad) = struct
```

Reply:
Unfortunately the `Monad` module types doesn't exist inside stdune :(

Reply:
Oh, indeed :(